src/legacy/ui/public/courier/fetch/components/_shard_failure_modal.scss

```diff
@@ -0,0 +1,50 @@
+// set width and height to fixed values to prevent resizing when you switch tabs
+.shardFailureModal {
+     height: 75vh;
+     width: 768px;
+}
+
+// displaying a EuiCodeBlock inside a tab inside a modal, you've to set the width 
+// to prevent overlapping the modal when the content contains very long lines
+.shardFailureModal__codeblock {
+    width: 702px
+}
+
+// making EuiDescriptionList a bit more compact, there's not so much space inside a modal
+.euiDescriptionList.euiDescriptionList--column {
```

Comment:
Try not to use `.eui` classes directly in overrides. You should be able to add a className directly to `EuiDescriptionList` and then use that class here.

Reply:
hmm, here I've some problems, I don't want to use important, my class gets overwritten by the EUI class
<img width="314" alt="Bildschirmfoto 2019-07-24 um 23 37 12" src="https://user-images.githubusercontent.com/463851/61830671-0976ea80-ae6c-11e9-9d55-52e0d3ea880f.png">

Reply:
It's better to use `!important` on a custom class than to write out the `.eui` class names. 

A. You'll never get out of sync. We _sometime_ do change class names.
B. Some newbie comes along and puts some styles before the next line (the custom class) and now all of sudden that style applies to **all** EuiDescriptionLists in Kibana.

Reply:
thx, so `important` it will be!